subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -6584,10 +6590,32 @@ static u32_t access_addr_get(void)
 static void adv_scan_conn_configure(void)
 {
 	radio_reset();
-	radio_tx_power_set(RADIO_TXP_DEFAULT);
+	radio_tx_power_set(_radio.tx_power_lvl);
 	radio_isr_set(isr, NULL);
 }
 
+void ll_tx_pwr_get(s8_t *min, s8_t *max)
+{
+	*min = (s8_t)radio_tx_power_min_get();
+	*max = (s8_t)radio_tx_power_max_get();
+}
+
+void ll_tx_pwr_set(const u32_t tx_power_lvl) 
+{
+	_radio.tx_power_lvl = tx_power_lvl;
+	radio_tx_power_set(_radio.tx_power_lvl);
```

Comment:
h/w registers should only be accessed in a controlled fashion, inside respect state/role's reserved time space (BLE radio events)